### PR TITLE
Initial Commit of Navbar

### PR DESCRIPTION
**Pull request by laxmikanthkonthum**
image

### CODE REVIEW

File: navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import url(/style.css);
+
+.nav-bar{
+    display: flex;
```
Comment (aniketsingh13) [inline navbar/navbar.css:4]: you can put flex in utility class if it used frequently

**Review by aniketsingh13 — APPROVED** [review]
consider the changes otherwise good to go

**Review by Creator-singh — APPROVED** [review]
Looks good to me

File: navbar/navbar.css (hunk 1 above)
Comment (laxmikanthkonthum) [reply navbar/navbar.css:4]: sure! will see in all the components and update them all together.

**Pull request merged**